examples/src/chain-question-answer.ts

```diff
@@ -29,30 +30,35 @@ export default async function run() {
 
   const openai = new OpenAI(apiKey);
 
+  const pt = new PromptTemplate("{{data}}");
```

Comment:
I'm not getting a type error when I build a prompt here

ex.
pt.build({}) doesn't show any type errors?